src/form.ts

```diff
@@ -43,7 +45,7 @@ export class Form {
         this.setActive(t)
         const len = this.fields.length
         const enabled = new Array(len).fill(false)
-        let current = 0
+        this.i = 0
```

Comment:
It'd be nicer to change this line to `let current = this.i`, add a `this.i = current` at the bottom and keep the rest of the code unchanged.

It'll also be nice to rename `this.i` to `this.currentField`